Snowplow/Internal/Emitter/SPEmitter.h

```diff
@@ -29,31 +29,11 @@
 #import <Foundation/Foundation.h>
 #import "SPNetworkConnection.h"
 #import "SPEventStore.h"
+#import "SPEmitterConfiguration.h"
 
 @protocol SPRequestCallback;
 @class SPPayload;
 
-/*!
- @brief An enum for buffer options.
- */
-typedef NS_ENUM(NSUInteger, SPBufferOption) {
-    /**
-     * Sends both GET and POST requests with only a single event.  Can cause a spike in
-     * network traffic if used in correlation with a large amount of events.
-     */
-    SPBufferOptionSingle = 1,
-    /**
-     * Sends POST requests in groups of 10 events.  This is the default amount of events too
-     * package into a POST.  All GET requests will still emit one at a time.
-     */
-    SPBufferOptionDefaultGroup = 10,
-    /**
-     * Sends POST requests in groups of 25 events.  Useful for situations where many events
-     * need to be sent.  All GET requests will still emit one at a time.
-     */
-    SPBufferOptionHeavyGroup = 25
```

Comment:
Oh I see this is existing naming now. Maybe a chance to change it but not required.